ui/src/shared/components/Dygraph.js

```diff
@@ -200,17 +211,61 @@ export default React.createClass({
       underlayCallback: options.underlayCallback,
       series: dygraphSeries,
     })
+    // if (this.lastMouseMoveEvent) {
+    //   dygraph.mouseMove_(this.lastMouseMoveEvent)
+    // }
 
     dygraph.resize()
-  },
+  }
+
+  sync() {
+    if (this.props.synchronizer && !this.state.synced) {
```

Comment:
Maybe this was written to protect future graphs, including `<MiniGraph>` (not used in the app currently) from being synced automatically. I'll change up the logic a bit.